mobile/lib/features/class_management/data/repositories/classroom_repository_impl.dart

```diff
@@ -38,13 +47,15 @@ class ClassroomRepositoryImpl implements ClassroomRepository {
 
   @override
   Future<Either<Failure, void>> deleteClassroom(Classroom classroom) async {
+    classroom = updateClientLastUpdated(classroom);
     return await _tryDeleteClassroom(classroom);
   }
 
   Future<Either<Failure, void>> _tryDeleteClassroom(Classroom classroom) async {
     try {
       var model = classroomEntityToModel(classroom);
       await localDataSource.deleteClassroomFromCache(model);
+      return null;
```

Comment:
We don't need to `return null` here.

[Avoid returning null for void](https://dart-lang.github.io/linter/lints/avoid_returning_null_for_void.html#:~:text=Dart%20allows%20returning%20null%20in,only%20use%20an%20empty%20return.)